docs/integrations/javascript/index.md

```diff
@@ -0,0 +1,36 @@
+# JavaScript
```

Comment:
in general: 1-line desc under each heading might be helpful, for example 'for client-side logging' or 'for edge worker environments' etc. WDYT?